packages/phoenix-client/src/phoenix/client/resources/spans/__init__.py

```diff
@@ -498,6 +501,51 @@ def log_spans_dataframe(
         spans = _dataframe_to_spans(spans_dataframe)
         return self.log_spans(project_identifier=project_identifier, spans=spans, timeout=timeout)
 
+    def delete_span(
```

Comment:
should just be `delete`
```suggestion
    def delete(
```